components/MashDashboard/MashDashboard.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import {
+  mockedMashReferral,
+  mockedMashReferrals,
+} from 'factories/mashReferral';
+import { ReferralStage } from 'types';
+import MashDashboard from './MashDashboard';
+
+let tabType: string | undefined;
+let confirmationType: string | undefined;
+const mockedRouter = {
+  query: { confirmation: confirmationType, tab: tabType },
+  push: jest.fn(),
+};
+
+jest.mock('next/router', () => ({
+  useRouter: () => mockedRouter,
+}));
+
+describe('MashDashboard', () => {
+  const contactMashReferral = mockedMashReferral;
+  contactMashReferral.stage = ReferralStage.CONTACT;
+  mockedMashReferrals.push(contactMashReferral);
+  const { contact, initial } = {
+    contact: mockedMashReferrals.filter(
+      (ref) => ref.stage === ReferralStage.CONTACT
+    ),
+    initial: mockedMashReferrals.filter(
+      (ref) => ref.stage === ReferralStage.INITIAL
+    ),
+  };
+
+  beforeEach(() => {
+    mockedRouter.query.tab = undefined;
+    mockedRouter.query.confirmation = undefined;
+  });
+
+  it('renders the dashboard default tab to be the contact tab', () => {
+    render(<MashDashboard referrals={mockedMashReferrals} />);
+
+    expect(contact.length).toBe(1);
```

Comment:
I#m not sure what the purpose of this expectation is, doesn't seem to test what the component is rendering but the logic within our test component (line 24-31).

Same applies to initial.length on line 51.

Reply:
Removed in 8b5297d